packages/access-api/src/routes/raw.js

```diff
@@ -17,9 +21,24 @@ export async function postRaw(request, env) {
     },
   })
 
+  const body = new Uint8Array(await request.arrayBuffer())
   const rsp = await server.request({
-    body: new Uint8Array(await request.arrayBuffer()),
+    body,
     headers: Object.fromEntries(request.headers.entries()),
   })
+
+  // Process CAR with invocations asynchronously
+  ctx.waitUntil(
```

Comment:
Should we store only successfully invoked requests? i.e. should we check `res.error`? Can we even? I think no right here.

I assume that later we'll receive a receipt that we can also store so we can know about failures?

cc @Gozala

Reply:
In the UCAN Stream we will see the contents of the `nb` and if it has an error we will skip it. But, if we can skip it here would be better for sure